src/SourceBuild/content/build.sh

```diff
@@ -226,6 +226,9 @@ function Build {
       # kill off the MSBuild server so that on future invocations we pick up our custom SDK Resolver
       "$CLI_ROOT/dotnet" build-server shutdown
 
+      # Point MSBuild to the custom SDK resolvers folder, so it will pick up our custom SDK Resolver
+      export MSBUILDADDITIONALSDKRESOLVERSFOLDER="$scriptroot/artifacts/toolset/VSSdkResolvers/"
```

Comment:
Which component in the orchestrator references the Arcade Sdk? I thought we disabled the Arcade SDK in the orchestration layer when building from source.

Reply:
Ah found the reason. It's /eng/tools, /src and /tests that don't import arcade but /repo-projects do.